### PR TITLE
Add a Quart view

### PR DESCRIPTION
**Pull request by catwell**
## Description

This PR adds a Quart view, as originally asked in [this issue](https://github.com/strawberry-graphql/strawberry/issues/149#issuecomment-671613711) but never actually implemented.

Quart 0.19 being a major release now depending on Flask 3 and Werkzeug 3, I had to bump the Flask support and remove pytest-flask which is incompatible (but AFAIK not required by Strawberry).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

This release adds a Quart view.



Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Pierre Chapuis for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

This release adds a Quart view.



Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Pierre Chapuis for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

File: noxfile.py [hunk 1]
```diff
@@ -93,10 +95,6 @@ def tests_integrations(session: Session, integration: str) -> None:
 
     if integration == "aiohttp":
         session._session.install("pytest-aiohttp")  # type: ignore
-    elif integration == "flask":
-        session._session.install("pytest-flask")  # type: ignore
-        # TODO: pytest-flask doesn't support Flask 3.0 yet
-        session._session.install("flask<3")  # type: ignore
```
Comment (patrick91) [inline noxfile.py:99]: thanks!

**Review by patrick91 — COMMENTED** [review]
This looks very nice! Thank you, I'll try it today or tomorrow and the release it 😊

**Review by patrick91 — APPROVED** [review]
Looks great!

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Pull request merged**